google-map-marker.html

```diff
@@ -366,7 +366,7 @@
         subtree: true
       });
 
-      var content = this.innerHTML.trim();
+      var content = this.children[0];
```

Comment:
@ebidel In my use case, I'm using `<google-map-marker><h2>...</h2><custom-element>...</custom-element>...</google-map-marker>` not using a single wrapper. Using just text works as well for me.

I'm afraid I'm a bit fuzzy on the shadow dom / light dom distinction or what use cases the method I'm using breaks.